apps/console/src/features/organizations/components/edit-organization-role/organization-role-permission.tsx

```diff
@@ -196,6 +197,7 @@ export const OrganizationPermissionList: FunctionComponent<OrganizationPermissio
          * the tree nodes.
          * @param eventObject - event object
          */
+        // eslint-disable-next-line @typescript-eslint/typedef
```

Comment:
Any specific reason for disabling typedef rule here?

Reply:
Same as this comment : https://github.com/wso2/identity-apps/pull/4212#discussion_r1359846958